core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -111,8 +112,12 @@ object FormattedBy extends Enum[FormattedBy] {
   final case object Custom extends FormattedBy {
     override def asString: String = "CUSTOM"
   }
+
+  final case object Cromwell extends FormattedBy {
+    override def asString: String = "CROMWELL"
+  }
 }
 
 final case class PvcId(asString: String) extends AnyVal
 // information needed for restoring a galaxy app
-final case class GalaxyRestore(galaxyPvcId: PvcId, cvmfsPvcId: PvcId, lastUsedBy: AppId)
+final case class GalaxyRestore(galaxyPvcId: PvcId, cvmfsPvcId: PvcId)
```

Comment:
I'd prefer the data model to be sth like
```
sealed trait AppRestore extends Product with Serializable {
  def lastUsedBy: AppId
}
object AppRestore {
  final case class GalaxyRestore(galaxyPvcId: PvcId, cvmfsPvcId: PvcId, lastUsedBy: AppId) extends AppRestore
  final case class CromwellRestore(lastUsedBy: AppId) extends AppRestore
}
```

Reply:
So with this is the idea that [PersistentDiskRecord](https://github.com/DataBiosphere/leonardo/blob/cdd26de1b56a92fa3005f28835b1a2d0edfa4596/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala#L33-L34) with store both GalaxyRestore and CromwellRestore details, and those details will be populated based on which app is present in `formattedBy` field?

Reply:
yea exactly...can jump on a call if you need some help with that

Reply:
I see. I will try and refactor it. If I come across any hurdles I can let you know. Thank you :)